packages/components/pie-input/src/input.scss

```diff
@@ -1 +1,11 @@
 @use '@justeattakeaway/pie-css/scss' as p;
+
+input[type="number"]:not([step])::-webkit-inner-spin-button,
```

Comment:
This is placeholder styling which will be replaced in the upcoming styling ticket. Please disregard it.

Reply:
To-do comment with ticket reference?